pkg/hubble/observer/local_observer.go

```diff
@@ -143,21 +142,21 @@ func (s *LocalObserverServer) Start() {
 	defer cancel()
 
 nextEvent:
-	for pl := range s.GetEventsChannel() {
+	for monitorEvent := range s.GetEventsChannel() {
 		for _, f := range s.opts.OnMonitorEvent {
-			stop, err := f.OnMonitorEvent(ctx, pl)
+			stop, err := f.OnMonitorEvent(ctx, monitorEvent)
 			if err != nil {
-				s.log.WithError(err).WithField("data", pl.Data).Info("failed in OnMonitorEvent")
+				s.log.WithError(err).WithField("event", monitorEvent).Info("failed in OnMonitorEvent")
 			}
 			if stop {
 				continue nextEvent
 			}
 		}
 
-		ev, err := s.payloadParser.Decode(pl)
+		ev, err := s.payloadParser.Decode(monitorEvent)
 		if err != nil {
-			if !parserErrors.IsErrInvalidType(err) {
-				s.log.WithError(err).WithField("data", pl.Data).Debug("failed to decode payload")
+			if err != parserErrors.ErrUnknownEventType || true {
```

Comment:
```suggestion
			if !errors.Is(err, parserErrors.ErrUnknownEventType) {
```
I assume that `|| true` is a leftover from local testing.

Reply:
:scream: true!

Reply:
Fixed